pkg/util/push/otel.go

```diff
@@ -45,6 +45,7 @@ func OTLPHandler(
 	maxRecvMsgSize int,
 	sourceIPs *middleware.SourceIPExtractor,
 	allowSkipLabelNameValidation bool,
+	EnableOtelMetadataStorage bool,
```

Comment:
The convention in Go is for parameter names to be camelCase.

```suggestion
	enableOtelMetadataStorage bool,
```

Reply:
fixed